### ISSUE TITLE
Design feedback - Usability Testing Prep - Healthcare [Questionnaires - Proof of Concept ]

### ISSUE THREAD

**Issue by CrystabelReiter** [department-of-veterans-affairs/va.gov-team#14686]
- **Must** for if the feedback must be applied
- **Should** if the feedback is best practice
- **Consider** for suggestions/enhancements

- If you have anything to submit to the design system council to review (issues with categories, labels) as well as the forms library header customization. Submit a ticket here: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/new/choose

- We noted in the meeting- some of these issues were caused by some UXpin wonkiness- please update anything that can be updated. 

**Must:**
- For [apply pages](https://preview.uxpin.com/a15205c1bedcdfc8fe46fed26c22fa731cab8ae0#/pages/133030703/simulate/no-panels?mode=cvhidm), make sure text style is H1 is in bitter 40px bold (on desktop) and 32px bold (on mobile) 
- For the lead paragraph “Better prepare yourself and your provider…”, make it the Lead text style bitter, regular, 20px
- Menu button in mobile prototype- the carot icon should be white 
- The icon in the info alerts, should be black and not turquoise (on the form apply and form review page)
- Instead of a continue button on the review page, have the button label be “Submit questionnaire” (defer to Peggy on verbiage)
- For the light green background alert that appears at the bottom of form pages, replace the info icon with a round checkmark icon
- For the [desktop version](https://preview.uxpin.com/e9e0dc597bb076e74db05f77d35306496a89fca2#/pages/133024274/simulate/no-panels?mode=cvhidm), the breadcrumbs should have less space between the webpages, see https://design.va.gov/components/breadcrumbs for example
- For the desktop review page, right align the bolded information (like how you have it on mobile) 
- For the [“Add Another Question” button on desktop](https://preview.uxpin.com/e9e0dc597bb076e74db05f77d35306496a89fca2#/pages/133049163/simulate/no-panels?mode=cvhidm), replace the disabled button state with the secondary button state like how you have it on mobile

**Should:**
- For the button, “Start your questionnaire”, we recommend the button to be green. If there is only one button on the page, use a green button. Especially if the button is surrounded by a lot of text. In most other cases, use a blue button. Green buttons are always alone, and meant to draw extra attention to themselves.
- For mobile apply page, widen the “Start the questionnaire” button to the width of the screen
- For the mobile form pages, align step progress tracker to the form content
Use the updated version of the step progress tracker in the sketch library. Instead of the number in the circle spell out the number ex: “Step 2 of 3…”
- For the [mobile review page](https://preview.uxpin.com/a15205c1bedcdfc8fe46fed26c22fa731cab8ae0#/pages/133030732/simulate/no-panels?mode=cvhidm), the “edit buttons” seem inconsistently sized, I’d double check and make sure the button size is uniform 
- “Continue and back” buttons on mobile should span to be a little bigger for easier clicking 
- Noticed on some buttons that labels aren’t centered on the button. 

**Consider:**
- For the subway map pattern, looks like step 2 and 3 are cut off (defer to Peggy for content feedback)
- Consider[ on this page](https://preview.uxpin.com/e9e0dc597bb076e74db05f77d35306496a89fca2#/pages/133049163/simulate/no-panels?mode=cvhidm), to shorten the text areas or lengthen the list loop cards to be uniform with each other 
- For the info alert and light green background alert on the desktop review page, have the length be the same as the review cards

**Definition of Done:**
- [x] Incorporate the **Must** feedback, and let us know on this ticket when they are updated mockups to check out 

- [x] Leave any comments for feedback you decide _not_ to take

Emily or Crystabel will close the ticket once those things are all set. Thank you!

**Comment by kristenmcconnell** [comment]
Everything is fixed, except for the following ones with questions or comments...

Must:
* For apply pages, make sure text style is H1 is in bitter 40px bold (on desktop) and 32px bold (on mobile)
    * I checked all the H1’s (Upcoming appointment questionnaire) on every page (screenshots included). They all seem to be correct in UXPin. Can you show me what you are seeing?
![Screen Shot 2020-10-11 at 9.42.51 PM.png](https://images.zenhubusercontent.com/5f44055c65a5c54f7cfe5928/686a706a-8ec2-4b31-899c-2713e8136cd2)
![Screen Shot 2020-10-11 at 9.43.04 PM.png](https://images.zenhubusercontent.com/5f44055c65a5c54f7cfe5928/93a47ca5-531b-40c1-920b-4641d80b2f6f)
* For the lead paragraph “Better prepare yourself and your provider…”, make it the Lead text style bitter, regular, 20px
    * It looks correct in my UXPin (screenshots included). Again, can you show me what you are seeing?
![Screen Shot 2020-10-11 at 9.48.40 PM.png](https://images.zenhubusercontent.com/5f44055c65a5c54f7cfe5928/153e1d02-5fb4-4edb-9a70-ae02f85d4e72)
* Instead of a continue button on the review page, have the button label be “Submit questionnaire” (defer to Peggy on verbiage)
    * Fixed. Peggy wants questions instead of questionnaire.
* For the desktop review page, right align the bolded information (like how you have it on mobile)
    * Interesting… my sketch file was correct. I’ll have to go back and look at Becca recommendations for UXPin, because how I imported definitely resulted in some weird things.
* For the “Add Another Question” button on desktop, replace the disabled button state with the secondary button state like how you have it on mobile
    * I thought it was disabled until the user types something into the first box, then it turns into the secondary button color. This is actually how I have desktop set up. I actually think that mobile should probably match my desktop design. Let me know if that is wrong and we truly have no disabled state in this instance.

Should:
* For the button, “Start your questionnaire”, we recommend the button to be green. If there is only one button on the page, use a green button. Especially if the button is surrounded by a lot of text. In most other cases, use a blue button. Green buttons are always alone, and meant to draw extra attention to themselves.
    * Changed. I have a question about the unauthenticated version of the page. According to https://dev-design.va.gov/patterns/benefit-applications, the top button is blue in this instance. What is the color of the button at the bottom of the page? Is it also blue or green (screenshot included)? If it isn’t green, what is the justification? Why is signed-in one way and signed-out another, even though they only have one real action on the page?
![Introduction - Reason for Visit - Desktop.jpg](https://images.zenhubusercontent.com/5f44055c65a5c54f7cfe5928/03b233ba-78b7-45b4-bc06-98b7df88c584)
Consider:
* For the subway map pattern, looks like step 2 and 3 are cut off (defer to Peggy for content feedback)
    * This is what I see? What do you see?
![Screen Shot 2020-10-12 at 4.01.45 PM.png](https://images.zenhubusercontent.com/5f44055c65a5c54f7cfe5928/9829d5ad-0ce0-4313-ae52-f6c0ec2c6a70)

**Comment by CrystabelReiter** [comment]
@kristenmcconnell 
For the typography feedback, I spoke to Becca today and she said that the font is actually correct but it just renders differently visually (it looks a little less bolded)- please disregard the "must" feedback having to do with typography 

- For the “Add Another Question” button on desktop, replace the disabled button state with the secondary button state like how you have it on mobile

  -  I thought it was disabled until the user types something into the first box, then it turns into the secondary button color. This is actually how I have desktop set up. I actually think that mobile should probably match my desktop design. Let me know if that is wrong and we truly have no disabled state in this instance.
   - Hm, I haven't seen it disabled on prod. Can you point me to where you've seen that example? If we want to keep the pattern consistent then I would make it the secondary button by default

Should:

- For the button, “Start your questionnaire”, we recommend the button to be green. If there is only one button on the page, use a green button. Especially if the button is surrounded by a lot of text. In most other cases, use a blue button. Green buttons are always alone, and meant to draw extra attention to themselves.
  - Changed. I have a question about the unauthenticated version of the page. According to https://dev-design.va.gov/patterns/benefit-applications, the top button is blue in this instance. What is the color of the button at the bottom of the page? Is it also blue or green (screenshot included)? I'm guessing that it is blue in this case too. Why is signed-in one way and signed-out another, even though they only have one real action on the page?
  - Yes the button would be blue for the unauthenticated page. I think we reserve blue for most actions (including sign in), but to actually start the application we reserve it for the green button. 

Consider:
- For the subway map pattern, looks like step 2 and 3 are cut off (defer to Peggy for content feedback)
   - This is what I see? What do you see?
   - When I reviewed it on Friday, I didn't see any words past "upcoming", in the screenshot it seems like Step 2 is cut off at "informat"

**Comment by kristenmcconnell** [comment]
For the “Add Another Question” button on desktop, our front-end engineer said that he pulled code straight from platform for the list loop (when I last talked to him...he is off today for the holiday) and it has a disabled state when I log into staging and look. However, I checked out the list loop in the disability form and there is no disabled state. I updated my wireframes to the secondary button styling as a result and I will talk through this more with engineering.

I believe the other two items are corrected, so I think that I addressed everything in this ticket.
